javalib/src/main/scala/java/net/SocketHelpers.scala

```diff
@@ -505,3 +511,19 @@ private[net] object ip6 {
     def IPV6_MULTICAST_HOPS: CInt = in.IPV6_MULTICAST_HOPS
   }
 }
+
+@extern
+object async {
```

Comment:
I think this should be `private[net]` also.  See the declaration just above.

In a better world, or at least more consistent world `SocketHelpers` would also be private.  I believe it is not here because something
in `unit-tests` reaches into `SocketHelpers` and executes a method.
No one, including me, has gotten around to removing that wart.